cli/internal/providers/datacatalog/state/property.go

```diff
@@ -31,7 +31,11 @@ func (args *PropertyArgs) DiffUpstream(upstream *catalog.Property) bool {
 		return true
 	}
 
-	return !reflect.DeepEqual(args.Config, upstream.Config)
+	upstreamConf := upstream.Config
+	if upstream.DefinitionId != "" {
+		upstreamConf = make(map[string]any)
```

Comment:
why are we setting upstreamConf as an empty map if the property has definitionId?